dipy/reconst/tests/test_csdeconv.py

```diff
@@ -564,8 +565,9 @@ def test_csd_superres():
                                               sh_order=16)
     fit16 = model16.fit(S)
 
+    sphere = HemiSphere.from_sphere(get_sphere('symmetric724'))
```

Comment:
Why are we still using the symmetric sphere here?

Reply:
To avoid to update the whole tests 😄. It seems that when we use repulsion, there is 3 peaks instead of 2 and they do not line up with the sticks